commands/parse.go

```diff
@@ -202,6 +265,12 @@ var ParseCommand = cli.Command{
 			Aliases: []string{"m", "mul"},
 			Value:   false,
 		},
+		&cli.BoolFlag{
+			Name:    "contest",
+			Usage:   "Indicates if this is a contest to parse. The number of problems should follow (Defaulted to 1). Example: egor parse --contest 5.",
```

Comment:
I would write: Indicates if the user is intending to parse a whole contest. If this flag is supplied, the user should provide the number of problems (Defaults to 1) that the contest have.